PowerShell Scanners/Mapped Printers/Mapped Printers.ps1

```diff
@@ -0,0 +1,30 @@
+$Printers = Get-WMIObject Win32_Printer | where{$_.Name -like "*\\*"}

```

Comment:
I believe this script can be simplified quite a bit.

```PowerShell
Get-CimInstance Win32_Printer | Where-Object Name -Like "\\*" | Select-Object ShareName, Name, Default
```

Reply:
Colby, you are correct. I have applied your suggestion and gave it a test and it works great.